src/rcamera.h

```diff
@@ -313,45 +313,44 @@ void UpdateCamera(Camera *camera)
             }
 
             // Camera looking down
-            else if ((camera->position.y > camera->target.y) && (CAMERA.targetDistance == CAMERA_FREE_DISTANCE_MAX_CLAMP) && (mouseWheelMove < 0))
+            else if ((camera->position.y > camera->target.y))
```

Comment:
`rcamera` module is being redesigned so no need to add changes at this moment, this file can be skipped